common/lib/xmodule/xmodule/html_module.py

```diff
@@ -372,6 +366,19 @@ def index_dictionary(self):
         return xblock_body
 
 
+def get_html(block: HtmlBlock):
```

Comment:
While this does work, it is not following the code organization conventions of this xmodules package. Can you make this a method in the `HtmlBlock`? Maybe it can be called `get_rendered_data()`.

Reply:
Actually I thought of a better way which mirrors what we had earlier: https://github.com/edx/edx-platform/pull/20954/files#diff-3ba53f005532d5ade732813b11c7911f480221810f6724433f6b5d0c568aa080L132

1. Rename the class `HtmlBlock` to `HtmlBlockMixin` (and its usage in the child class declarations).
2. Create a new class:

```
@edxnotes	
class HtmlBlock(HtmlBlockMixin):	
  pass
```

Now the `edxnotes` decorator will only apply to this XBlock class and will not interfere with the other ones.

Reply:
Genius, this is a great idea!